### PR TITLE
Added windows-package

### PR DESCRIPTION
**Pull request by monalivadje**

### CODE REVIEW

**Comment by JoyMonteiro** [comment]
Please add the python-package file back to the repository. It looks like github can't accept the pull request otherwise.

File: .github/workflows/windows-package.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  pull_request:
+    branches: [ master, develop ]
+
+name: windows
+jobs:
+  windows:
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        compiler: [gcc, gcc-fortran]
```
Comment (JoyMonteiro) [inline .github/workflows/windows-package.yml:17]: omit this, we need both compilers to be installed, not two different runs with different compilers

**Comment by monalivadje** [comment]
Ok

File: .github/workflows/windows-package.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    defaults:
+      run:
+        shell: msys2 {0}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: msys2/setup-msys2@v2
+      with:
+        install: >-
+          gcc make
```
Comment (JoyMonteiro) [inline .github/workflows/windows-package.yml:26]: delete next three lines, this line should be
`gcc make gcc-fortran winpthreads-git openblas`

File: .github/workflows/windows-package.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+          gcc-fortran make
+          make mingw-w64-i686-winpthreads-git
+          make mingw-w64-i686-openblas
+    - run: pacman --noconfirm -S gcc
```
Comment (JoyMonteiro) [inline .github/workflows/windows-package.yml:30]: I think the previous command will install gcc etc., automatically, so you don't need to run pacman separately. You can delete all `pacman` commands

File: .github/workflows/windows-package.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      if: ${{ matrix.compiler == 'gcc' }}
+    - run: pacman --noconfirm -S gcc-fortran 
+      if: ${{ matrix.compiler == 'gcc-fortran' }}
+    - run: ${{ matrix.compiler }} --version
+    - run: which gcc-fortran -v
+    - run: make CC=${{ matrix.compiler }}
```
Comment (JoyMonteiro) [inline .github/workflows/windows-package.yml:36]: delete, there is nothing to `make` currently

File: .github/workflows/windows-package.yml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+          gcc make
+          gcc-fortran make
+          make mingw-w64-i686-winpthreads-git
+          make mingw-w64-i686-openblas
+    - run: pacman --noconfirm -S gcc
+      if: ${{ matrix.compiler == 'gcc' }}
+    - run: pacman --noconfirm -S gcc-fortran 
+      if: ${{ matrix.compiler == 'gcc-fortran' }}
+    - run: ${{ matrix.compiler }} --version
+    - run: which gcc-fortran -v
```
Comment (JoyMonteiro) [inline .github/workflows/windows-package.yml:35]: no need of `-v`

File: .github/workflows/windows-package.yml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+        install: >-
+          gcc make
+          gcc-fortran make
+          make mingw-w64-i686-winpthreads-git
+          make mingw-w64-i686-openblas
+    - run: pacman --noconfirm -S gcc
+      if: ${{ matrix.compiler == 'gcc' }}
+    - run: pacman --noconfirm -S gcc-fortran 
+      if: ${{ matrix.compiler == 'gcc-fortran' }}
+    - run: ${{ matrix.compiler }} --version
```
Comment (JoyMonteiro) [inline .github/workflows/windows-package.yml:34]: run both `gcc --version` and `gfortran --version` separately

File: .github/workflows/windows-package.yml [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+      with:
+        install: >-
+          gcc make
+          gcc-fortran make
+          make mingw-w64-i686-winpthreads-git
+          make mingw-w64-i686-openblas
+    - run: pacman --noconfirm -S gcc
+      if: ${{ matrix.compiler == 'gcc' }}
+    - run: pacman --noconfirm -S gcc-fortran 
+      if: ${{ matrix.compiler == 'gcc-fortran' }}
```
Comment (JoyMonteiro) [inline .github/workflows/windows-package.yml:33]: remove

**Review by JoyMonteiro — CHANGES_REQUESTED** [review]

File: .github/workflows/windows-package.yml [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+name: windows package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master, develop ]
+
+name: windows
```
Comment (JoyMonteiro) [inline .github/workflows/windows-package.yml:11]: There are two `name` commands, remove one

**Pull request merged**